hdr_html_canvas_element.md

```diff
@@ -125,13 +125,23 @@ The component signals are mapped to red, green and blue tristimulus values accor
 * Transfer function:
 
 ```
-   E = | E' / 12.92, if abs(E') ≤ 0.04045
-       | ((E' + 0.055) / 1.055)^2.4, otherwise
```

Comment:
I'd prefer it to be

```
   E = | E' / 12.92, if abs(E') ≤ 0.04045
       | sign(E') * ((abs(E') + 0.055) / 1.055)^2.4, otherwise
```

That emphasizes the point symmetry better, in my opinion.